MfciPkg/MfciDxe/MfciDxe.inf

```diff
@@ -55,7 +55,7 @@
 
 
 [Protocols]
-  gMfciProtocolGuid                     ## CONSUMES
+  gMfciProtocolGuid                     ## PRODUCES CONSUMES

```

Comment:
I will note that `## PRODUCES ## CONSUMES` is also acceptable.

Reply:
Thanks for clarifying. I was searching the code base to try to find an example, and the only instance I found was without the second ##